proof/courier.go

```diff
@@ -313,7 +314,8 @@ func (h *HashMailCourier) DeliverProof(ctx context.Context, addr address.Taro,
 	log.Infof("Sending receiver proof via sid=%x", senderStreamID)
 	err := h.mailbox.WriteProof(ctx, senderStreamID, proof.Blob)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to send proof to asset transfer"+
```

Comment:
nit: would be nice to have this uniform to make scanning with the eye easier: space between words should be before the break.